working/templates/package-project/GettingStarted.md

```diff
@@ -0,0 +1,154 @@
+2. If you don’t require `.dmapp` creation, review your project setup to ensure it aligns with your goals.
+<!--#endif-->
+
+<!--#if (IsCatalogNoCICD)-->
+## Publishing to the Catalog
+
+This project was created with support for publishing to the DataMiner catalog.  
+You can publish your artifact manually through Visual Studio or by setting up a CI/CD workflow.
+
+### Manual Publishing
+
+1. Obtain an **Organization Key** from [admin.dataminer.services](https://admin.dataminer.services/) with the following scopes:
+   - **Register catalog items**
+   - **Read catalog items**
+
+2. Securely store the key using Visual Studio User Secrets:
+   - Right-click the project and select **Manage User Secrets**.
+   - Add the key in the following format:
+
+```json
+{
+  "skyline": {
+    "sdk": {
+      "catalogpublishtoken": "MyKeyHere"
+    }
+  }
+}
+```
+
+3. Publish the package using the **Publish** option in Visual Studio.
+
+**Recommendation:** For stable releases, consider using a CI/CD setup to run **dotnet publish** after passing quality checks.
+
+<!--#elseif (IsCatalogBasicCICD)-->
+## Publishing to the Catalog with Basic CI/CD Workflow
+
+This project includes a basic GitHub workflow for catalog publishing.  
+Follow these steps to set it up:
+
+1. Create a GitHub repository and push your project.
+2. The first workflow run may fail due to missing secrets. View the error details on the **Actions** page of your repository.
+3. Obtain an **Organization Key** from [admin.dataminer.services](https://admin.dataminer.services/) with the following scopes:
+   - **Register catalog items**
+   - **Read catalog items**
+
+4. Add the key as a secret in your GitHub repository:
+   - Navigate to **Settings > Secrets and variables > Actions** and create a secret named `DATAMINER_TOKEN`.
+5. Re-run the workflow.
+
+With this setup, any push to the main/master branch will generate a new pre-release version, using the latest commit message as the version description.
+
+### Releasing a Specific Version
+
+- Navigate to the **<> Code** tab in your repository.
+- Select **Releases** from the right-hand menu.
+- Draft a new release, select the desired version, and provide a description.
+
+<!--#elseif (IsCatalogCompleteCICD)-->
+## Publishing to the Catalog with Complete CI/CD Workflow
+
+This project includes a comprehensive GitHub workflow that adheres to Skyline Communications' quality standards, including static code analysis, custom validation, and unit testing.
+
+### Prerequisite:  
+You need a **SonarCloud Organization**. If you don’t have one, you can create it [here](https://sonarcloud.io/create-organization).
+
+### Steps:
+1. Create a GitHub repository and push your project.  
+   Note: For free SonarCloud accounts, your repository must be public.
+
+2. The first workflow run may fail due to missing secrets and variables. Follow the error messages on the **Actions** tab to resolve each issue and re-run the workflow multiple times as needed.
```

Comment:
```suggestion
2. If the first workflow run fails because of missing secrets and variables, check the error messages on the **Actions** tab to resolve each issue and re-run the workflow multiple times as needed.
```